HLTrigger/Configuration/python/HLT_75e33/modules/me0RecHits_cfi.py

```diff
@@ -1,9 +0,0 @@
-import FWCore.ParameterSet.Config as cms
-
-me0RecHits = cms.EDProducer("ME0RecHitProducer",
-    me0DigiLabel = cms.InputTag("simMuonME0PseudoReDigis"),
-    recAlgo = cms.string('ME0RecHitStandardAlgo'),
-    recAlgoConfig = cms.PSet(
-
-    )
-)
```

Comment:
In `HLTrigger/Configuration/python/HLT_75e33_cff.py`
https://github.com/cms-sw/cmssw/blob/2f899a28884ae473969923fb54dc5478245f3098/HLTrigger/Configuration/python/HLT_75e33_cff.py#L317-L318

I still see
```
del fragment.me0RecHits
del fragment.me0Segments
```
Should this become .. ?
```
del fragment.hltMe0RecHits
del fragment.hltMe0Segments
```
(I guess it should be `ME0`, rather than `Me0`, but that's a detail.)